nextflow.config

```diff
@@ -190,9 +191,11 @@ profiles {
         apptainer.enabled       = false
         docker.runOptions       = '-u $(id -u):$(id -g)'
     }
-    arm {
+    apple {
         docker.runOptions       = '-u $(id -u):$(id -g) --platform=linux/amd64'
-        includeConfig 'conf/arm.config'
+    }
+    arm {
```

Comment:
As you mentioned in https://github.com/nf-core/tools/issues/3536, this should be renamed but with a different name, to avoid confusion.
The rest looks good :)